rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -258,51 +279,93 @@ The credential can be deployed in the production phase.
 
 * *Type*:
 +
-Type of the global credential. The type of credential determines where it can be used. The original value defined in RPA Builder is typed. Select one of the following types:
+Type of the global credential. The type of credential determines where you can use it. The original value defined in RPA Builder is typed. Select one of the following types:
 +
 ** *AWS Credentials*:
 +
-The credential is used for accessing Amazon Web Services. Fill in the client ID, client secret and region. Using a session token is optional.
+Use this credential for accessing Amazon Web Services. Fill in the client ID, client secret, and region. Using a session token is optional.
+** *OAuth Credentials*
++
+Use this credential for accessing the Microsoft Outlook email application via Microsoft Azure.
++
+*** *OAuth Host*:
++
+The host address used to get the credentials. The default address for Outlook services is https://login.microsoftonline.com/common/oauth2/v2.0/ . This address might change if the customer uses a self hosted service.
```

Comment:
There seems to be an extra space at:  "The default address for Outlook services is https://login.microsoftonline.com/common/oauth2/v2.0/` `."

Reply:
Added `` because it is a value the user needs to enter, not a link to click on.